eland/ml/ml_model.py

```diff
@@ -424,6 +435,63 @@ def exists_model(self) -> bool:
             return False
         return True
 
+    def export_model(self) -> "Pipeline":
+        """Export Elastic ML model as sklearn Pipeline.
+
+        Returns
+        -------
+        sklearn.pipeline.Pipeline
+            _description_
+
+        Raises
+        ------
+        ValueError
+            The model is expected to be trained in Elastic Stack. Models initially imported
+            from xgboost, lgbm, or sklearn are not supported.
+        NotImplementedError
+            Only regression and binary classification models are supported currently.
+        """
+        from sklearn.compose import ColumnTransformer  # type: ignore # noqa: F401
+        from sklearn.pipeline import Pipeline
+
+        if self.model_type == TYPE_CLASSIFICATION:
+            model = ESGradientBoostingClassifier(
+                es_client=self._client, model_id=self._model_id
+            )
+        elif self.model_type == TYPE_REGRESSION:
+            model = ESGradientBoostingRegressor(
+                es_client=self._client, model_id=self._model_id
+            )
+        else:
+            raise NotImplementedError
+
+        if "preprocessors" in model._definition:
+            preprocessors = model._definition["preprocessors"]
+        else:
+            preprocessors = []
+        transformers = []
+        for p in preprocessors:
+            encoding_type = list(p.keys())[0]
```

Comment:
Since this is assuming a one-key mapping can we add an assertation of `len(p) == 1`?

Reply:
Assertion added